[
 {
  "author": "rajith", 
  "date": "2011-05-03T14:04:46.671678Z", 
  "msg": [
   "QPID-3214", 
   "Committing the suggested fix by Gordon Sim.", 
   "All though the currentException may change by the time we call connection.exceptionReceived() it's not really a big issue.", 
   "Chance of getting two successive sessions exceptions is rare if not none at all."
  ], 
  "revision_id": "1099060"
 }, 
 {
  "author": "aadamchik", 
  "date": "2011-05-03T14:04:04.161445Z", 
  "msg": [
   "CAY-1563 CayenneCase unit tests conversion(3)", 
   "patches by Dzmitry Kazimirchyk:", 
   "DeleteRulesTest"
  ], 
  "revision_id": "1099059"
 }, 
 {
  "author": "mturk", 
  "date": "2011-05-03T14:03:33.798307Z", 
  "msg": [
   "Set both server sockets to nonblockin mode for TwoStacks sockets"
  ], 
  "revision_id": "1099058"
 }, 
 {
  "author": "aadamchik", 
  "date": "2011-05-03T13:59:36.267458Z", 
  "msg": [
   "CAY-1563 CayenneCase unit tests conversion(3)", 
   "patches by Dzmitry Kazimirchyk:", 
   "DeleteBatchQueryBuilderTest"
  ], 
  "revision_id": "1099057"
 }, 
 {
  "author": "aadamchik", 
  "date": "2011-05-03T13:55:07.182194Z", 
  "msg": [
   "CAY-1563 CayenneCase unit tests conversion(3)", 
   "patches by Dzmitry Kazimirchyk:", 
   "DefaultOperationObserverTest"
  ], 
  "revision_id": "1099056"
 }
]